packages/components/src/components/video-player/video-player.scss

```diff
@@ -0,0 +1,285 @@
+                    opacity: 0;
+                }
+                &--enter-done {
+                    transform: translateY(0);
+                    opacity: 1;
+                }
+            }
+
+            &__filled {
+                position: absolute;
+                bottom: 0;
+                width: 0.4rem;
+                border-radius: $BORDER_RADIUS;
+                background-color: var(--brand-red-coral);
+
+                &--animated {
+                    transition: all 0.3s linear;
+                }
+            }
+        }
+
+        &-dot {
+            position: absolute;
+            left: -0.4rem;
+            bottom: 90%;
+        }
+    }
+
+    &__progress-dot,
+    &__volume-dot {
+        width: 1.2rem;
+        height: 1.2rem;
+        border-radius: 50%;
+        background-color: var(--brand-red-coral);
+        cursor: pointer;
+    }
+
+    &__progress-dot {
+        position: absolute;
+        left: calc(100% - 0.6rem);
+        bottom: -0.3rem;
+
+        @include desktop {
+            &:hover {
+                bottom: -0.6rem;
+                width: 1.6rem;
+                height: 1.6rem;
+            }
+        }
+
+        &--enlarged {
+            bottom: -0.6rem;
+            width: 1.6rem;
+            height: 1.6rem;
+        }
+    }
+
+    &__playback-rate__wrapper {
+        width: 6.2rem;
+        position: relative;
+
+        .playback-rate__icon {
+            position: absolute;
+            left: 0;
+        }
+
+        .dc-dropdown {
+            &__select-arrow {
+                display: none;
+            }
+
+            &__display--playback-rate {
+                width: 6.2rem;
+                height: 2.3rem;
+                background: none;
+                border: none;
+                border-radius: $BORDER_RADIUS * 2;
+                justify-content: flex-start;
+
+                .dc-dropdown__display-text {
+                    padding: 0 0 0 2.5rem;
+                    color: var(--text-colored-background);
+                    font-size: 1rem;
+                }
+            }
+
+            &-container {
+                min-width: 6.2rem;
+                width: unset;
+
+                &:before {
+                    content: '';
+                    display: block;
+                    height: 1.7rem;
+                    width: 7rem;
+                    position: absolute;
+                    bottom: 1rem;
+                }
+            }
+            &__list {
+                width: 9.7rem;
+                right: 10%;
+                left: unset;
+            }
+
+            &__display--playback-rate__item {
+                height: 4rem;
+                display: flex;
+                justify-content: start;
+                align-items: center;
+            }
+        }
+
+        .dc-list,
+        .dc-themed-scrollbars {
+            @include mobile {
+                height: 13.2rem;
+            }
+        }
+    }
+}
```

Comment:
if you are completely moving the video-player from trader to components, could you please remove the remaining unused duplicate from trader package?  ) if it's no longer used of course